### PR TITLE
Feat/husky

### PR DESCRIPTION
**Pull request by DeyvidHolz**
## Status
**READY**

## Migrations
NO

## Description
Add husky and commitlint

## Type of change
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Steps to Test or Reproduce
Commit/Push something and make sure husky is applying

## Checklist:
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,5 +66,10 @@
     },
     "coverageDirectory": "../coverage",
     "testEnvironment": "node"
+  },
+  "husky": {
```
Comment (luqezman) [inline package.json:70]: I prefer using separated files to prevent growth in package.json

You can remove this section.

File: package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@commitlint/config-conventional": "^9.1.1",
```
Comment (luqezman) [inline package.json:24]: This package should be installed as dev dependencies

File: package.json (hunk 1 above)
Comment (DeyvidHolz) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (DeyvidHolz) [reply package.json]: done

**Pull request merged**